src/routes/canvas/toolbar/DrawingManager.tsx

```diff
@@ -98,12 +102,17 @@ class DrawingManager extends React.Component<{inkingManager: InkingManager}> {
         }
 
         return (
-            <div>
-                <Pen {...toolProps} />
-                <Highlighter {...toolProps} />
-                <Eraser {...toolProps} />
-                <LaserPointer {...toolProps} />
+            <div className='tool-second-level' style={{display: this.props.display}}>
+                {/* <FluentProvider theme={webLightTheme}> */}
+                <Toolbar className='popover'>
+                    <Pen {...toolProps} icon={ <img src={require("../../../assets/ink-pen.png")} alt="Icon" />}/>
```

Comment:
The requiring of the assets could be moved to the beginning of the files after the imports. Or even better each tool could require their icon directly in their own file.